spec/index.bs

```diff
@@ -694,25 +694,25 @@ A [=physical resource=] can be used on GPU in one of the following <dfn dfn>inte
 <dl dfn-type=dfn dfn-for="internal usage">
     : <dfn>input</dfn>
     :: Buffer with input data for draw or dispatch calls. Preserves the contents.
-        Allowed by {{GPUBufferUsage/INDEX}}, {{GPUBufferUsage/VERTEX}}, or {{GPUBufferUsage/INDIRECT}}.
+        Allowed by buffer {{GPUBufferUsage/INDEX}}, buffer {{GPUBufferUsage/VERTEX}}, or buffer {{GPUBufferUsage/INDIRECT}}.
```

Comment:
I believe we generally write this as `{{GPUBufferUsage/INDEX|GPUBufferUsage.Index}}` instead